games/godot/Portfile

```diff
@@ -22,24 +22,52 @@ long_description    Godot Engine is a cross-platform game engine for \
                     (Linux, macOS, Windows), as well as mobile \
                     (Android, iOS) and web-based (HTML5) platforms.
 
-checksums           rmd160  4ff4f9c4798d67cf6cf4d63c38bed84d400daa8e \
+if {$subport eq ${name}} {
+    github.setup    godotengine ${name} 3.3 "" -stable
+    version         3.3.0
+
+    checksums       rmd160  4719f38a85cf339fa6c66d5849b972cd22c0a1ed \
+                    sha256  a65e296c900f442e19600853896f8f742a1414cc8864f9f6e45eeabc9c4b3362 \
+                    size    29432959
+
+    compiler.blacklist-append {clang < 802}
+}
+
+subport ${name}-3.2 {
+    github.setup    godotengine ${name} 3.2.3 "" -stable
+
+    checksums       rmd160  4ff4f9c4798d67cf6cf4d63c38bed84d400daa8e \
                     sha256  6cd925cc4c23e2c4b0bf36458a83e03484184d8d5311b8eb2339319df7c0c26c \
                     size    22460848
 
-patchfiles          legacy-osx-defines.diff
+    long_description-append \
+                    \n\n(Note: Godot ${version} is the last version of \
+                    Godot that is officially supported on macOS \
+                    10.9-10.11.)
+}
+
+if {${os.major} <= 11} {
```

Comment:
this should be preceded by a check for `os.platform` as in ```if {${os.platform} eq "darwin" && ${os.major} <= 11} {``` (and perhaps use `< 12`?)

Reply:
So, I'm curious about this bit. Doesn't the fact that I have `platforms darwin` at the top of the file already essentially mean that the entire Portfile is for `${os.platform} eq "darwin"`? Or am I misunderstanding the purpose of `platforms darwin`?

Reply:
Completed requested change